
# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1450036112 18000
# Node ID bcb6edf35d93c85796321e7fe8fdcf5dd0ba09ac
# Parent  05125a9eb9c61c6a7f12e6dea07cac4ceb4b72e1
Add missing semicolon (from changeset c7d881aec36c)
(bug #42838).

* scripts/plot/util/__gnuplot_drawnow__.m: Add missing semicolon.

diff --git a/scripts/plot/util/__gnuplot_drawnow__.m b/scripts/plot/util/__gnuplot_drawnow__.m
--- scripts/plot/util/__gnuplot_drawnow__.m
+++ scripts/plot/util/__gnuplot_drawnow__.m
@@ -270,7 +270,7 @@
     term_str = ["set terminal " term];
     if (__gnuplot_has_feature__ ("needs_color_with_postscript") ...
         && strcmp (term, "postscript"))
-      term_str = [term_str, " color"]
+      term_str = [term_str, " color"];
     endif
     if (! isempty (enh_str))
       term_str = [term_str " " enh_str];

